This disables a couple of tests that fail on debci but not on my local system,
i'm pretty sure these are failing due to permission issues rather than bugs
in the code -- plugwash.

It also removes the "test_as_root" feature which enables a bunch of tests that
are unlikely to function in a CI environment.

Index: rtnetlink/src/traffic_control/add_filter.rs
===================================================================
--- rtnetlink.orig/src/traffic_control/add_filter.rs
+++ rtnetlink/src/traffic_control/add_filter.rs
@@ -357,8 +357,8 @@ mod test {
         }
     }
 
-    #[test]
+    /*#[test]
     fn test_new_filter() {
         Runtime::new().unwrap().block_on(test_async_new_filter());
-    }
+    }*/
 }
Index: rtnetlink/src/traffic_control/add_qdisc.rs
===================================================================
--- rtnetlink.orig/src/traffic_control/add_qdisc.rs
+++ rtnetlink/src/traffic_control/add_qdisc.rs
@@ -195,8 +195,8 @@ mod test {
         }
     }
 
-    #[test]
+    /*#[test]
     fn test_new_qdisc() {
         Runtime::new().unwrap().block_on(test_async_new_qdisc());
-    }
+    }*/
 }
Index: rtnetlink/Cargo.toml
===================================================================
--- rtnetlink.orig/Cargo.toml
+++ rtnetlink/Cargo.toml
@@ -97,7 +97,6 @@ smol_socket = [
     "netlink-proto/smol_socket",
     "async-global-executor",
 ]
-test_as_root = []
 tokio_socket = [
     "netlink-proto/tokio_socket",
     "tokio",
